static/app/components/teamRoleSelect.tsx

```diff
@@ -0,0 +1,91 @@
+import styled from '@emotion/styled';
```

Comment:
This component was extracted from `static/app/views/settings/organizationTeams/teamMembersRow.tsx` and generalized.